frontend/src/components/home/home.module.scss

```diff
@@ -1,6 +1,5 @@
 .home {
   display: grid;
-  grid-template:
-    'sidebar main'
-    /25% auto;
+  grid-template-areas: 'sidebar main';
+  grid-template-columns: auto 1fr;
```

Comment:
Instead of having sidebar width to auto and setting min and max width with sidebar's css, we can use the css minmax here directly
`grid-template-columns: minmax(200px, max-content) 1fr;`

Reply:
I think I'd tried that but there are issues when the width would be less than 200px. I had gone through a few iterations but the minmax stuff works a bit weirdly where if max-content < 200px, it ignores the max part and keeps it as 200px always. 

Switching them over (to `minmax(max-content, 200px) 1fr` ) doesn't work as intended either, although it was the closest I got. When width was between max-content and 200, the sidebar was about 50% of the view, not 25%. And messing around with nested minmax statements wasn't too helpful to me (probably because it got more confusing).

I'd spent some time trying to mess around with it, trying to keep within the grid style stuff, and with everything I tried other than `auto`, the `sidebar` grid cell never properly resized. (having a max width, min width and 25% width in between).